content/graphql/mapped-types.md

```diff
@@ -1,14 +1,16 @@
 ### Mapped types
 
-##### This chapter applies only to code first approach
+> warning **Warning** This chapter applies only to the code first approach.
 
-A common task is to take an existing type and make each of its properties optional. This happens often enough that NestJS provides a way to create new types based on old types — mapped types.
+As you build out features like CRUD (Create/Read/Update/Delete) it's often useful to construct variants on a base entity type. Nest provides several utility functions that perform type transformations to make this task more convenient.
 
-#### Partial
+#### Partial: PartialType() function
```

Comment:
I'm not sure, but I feel like `PartialType()` looks somewhat weird as part of the heading

Reply:
Yeah, I had mixed feelings too.  Figured I'd put it out there and see how you reacted :)  I'll come up with something better...